swift_scality_backend/diskfile.py

```diff
@@ -98,6 +98,14 @@ def write(self, chunk):
         self._md5sum.update(chunk)
         return self._upload_size
 
+    def chunks_finished(self):
```

Comment:
Is it used somewhere? By the tests/client ?

Reply:
This is called by the swift code itself, causing an exception if it is not implemented